redisinsight/ui/src/pages/workbench/components/wb-results/WBResults/WBResults.tsx

```diff
@@ -86,6 +90,16 @@ const WBResults = (props: Props) => {
           resultsMode={resultsMode}
           db={db}
           onQueryOpen={() => onQueryOpen(id)}
+          onQueryProfile={(profileType: ProfileQueryType) => {
+            const profileQuery = generateProfileQueryForCommand(command, profileType)
+            if (profileQuery) {
+              return onQueryProfile(
+                profileQuery,
+                null,
+                { mode, results: resultsMode, clearEditor: false, },
+              )
+            }
+          }}
```

Comment:
please move this function to separate one:
`onQueryProfile={handleQueryProfile}`
```
const handleQueryPrifile = (profileType: ProfileQueryType) => {...}
```